tools/cockpit.spec

```diff
@@ -53,6 +53,11 @@ Version:        0
 Release:        1%{?dist}
 Source0:        https://github.com/cockpit-project/cockpit/releases/download/%{version}/cockpit-%{version}.tar.xz
 
+# Experimental Python support
```

Comment:
The commit message still says "We move cockpit-askpass from /usr/bin to /usr/libexec". That's no longer true for the spec, as Makefile already does it.

Reply:
I dropped that paragraph.